server/migrations/20200915203422-AddColumnsToTestRun.js

```diff
@@ -26,19 +26,21 @@ module.exports = {
                 ),
                 queryInterface.addColumn(
                     'run',
-                    'createdAt',
+                    'created_at',
                     {
                         type: Sequelize.DATE,
-                        allowNull: false
+                        allowNull: false,
+                        defaultValue: Number.NEGATIVE_INFINITY
```

Comment:
unfortunately, new entries now have negative infinity for this value as well!

Reply:
@spectranaut oh dear! Can you show me how you tested what happens with new entries?

Reply:
yes def! I test by running the migration, then running "yarn dev", then adding a test cycle through the front end, then inspecting the `run` table in the database.

Reply:
I just tested out of curiosity, changing "defaultValue" to "Sequelize.Now" didn't work --- you still got the null constraints on the migration. I think we have no choice but to allow null. However, I think we also need to add Sequelize.Now to get those values populated. This means, in the migration:
```
                queryInterface.addColumn(
                    'run',
                    'created_at',
                    {
                        type: Sequelize.DATE,
                        defaultValue: Sequelize.NOW
                    },
                    { transaction: t }
                ),
                queryInterface.addColumn(
                    'run',
                    'updated_at',
                    {
                        type: Sequelize.DATE,
                        defaultValue: Sequelize.NOW
                    },
                    { transaction: t }
                )
```
and in the model:
```
            createdAt: {
                type: DataTypes.DATE,
                allowNull: true,
                defaultValue: sequelize.NOW
            },
            updatedAt: {
                type: DataTypes.DATE,
                allowNull: true,
                defaultValue: sequelize.NOW
            }
```

Reply:
@spectranaut You commit works! I suppose having nulls is just as informative as having something like a negative infinity take the place of a null.